packaging/conda/build_vision.sh

```diff
@@ -15,11 +15,44 @@ if [ "$#" -ne 1 ]; then
     echo "CUDA version should be M.m with no dot, e.g. '8.0' or 'cpu'"
     exit 1
 fi
+
```

Comment:
Heads up: this script is no longer used by the Linux/OS X binary builds (I didn't delete it because Windows still needs it), instead `packaging/build_conda.sh` is what is used. It would pretty awesome if we could also unify these packaging scripts, but I only really want to do it if we can keep the packaging on the shorter side. In the meantime, you can rename / simplify this script so that it is Windows only.